main.go

```diff
@@ -61,7 +61,18 @@ func exit(err error, exit int) {
 func main() {
 	runtime.GOMAXPROCS(runtime.NumCPU())
 
-	app := cli.App("mc-agent", "SSH agent that use moolticute to store/load your keys")
+	app := cli.App("mc-agent", `SSH agent that uses moolticute to store/load your keys
+
+An easy way to setup is to enable the "Autostart SSH Agent" setting in the Moolticute app, and set
+the "Moolticute SSH Arguments" to something like:
+
+  --address /tmp/mc-agent.socket
```

Comment:
Actually this is only true for macos or linux. On windows it uses putty emulation. So i'm not sure if we need to write this in the help command line here... The --address option is already shown with --help